test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/CryptoTransferHTSSuite.java

```diff
@@ -1538,6 +1568,1104 @@ private HapiApiSpec activeContractInFrameIsVerifiedWithoutNeedForSignature() {
                                                                 2L))));
     }
 
+    private HapiApiSpec hapiTransferFromForNFTWithCustomFeesWithoutApproveFails() {
+        return defaultHapiSpec("HapiTransferFromForNFTWithCustomFeesWithoutApproveFails")
+                .given(
+                        newKeyNamed(MULTI_KEY),
+                        newKeyNamed(RECEIVER_SIGNATURE),
+                        cryptoCreate(TOKEN_TREASURY),
+                        cryptoCreate(OWNER)
+                                .balance(100 * ONE_MILLION_HBARS)
+                                .maxAutomaticTokenAssociations(5)
+                                .key(MULTI_KEY),
+                        cryptoCreate(SENDER).balance(10 * ONE_HUNDRED_HBARS),
+                        cryptoCreate(RECEIVER)
+                                .balance(100 * ONE_MILLION_HBARS)
```

Comment:
There have been cases where our CI payer account has run out of money due to lots of high-balance accounts being created, so we might want to be less generous here. 😁

Reply:
Agree with you! :) DONE